### PR TITLE
Docker inspect command now shows sizes if request.

### PR DESCRIPTION
**Pull request by AE9999**
As I understand the docker command line interface, you can request the current size of an Image. I did not see this functionality in your API, but needed it for a project and have added it myself. Please feel free to copy it & thanks for your hard work :-)

Best wishes, 

AE

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/572)

### CODE REVIEW

**Comment by marcuslinke** [comment]
@AE9999 Thanks for adding this. Would it be possible for you to add a test case please?

**Comment by AE9999** [comment]
Will do

File: src/main/java/com/github/dockerjava/api/command/InspectContainerCmd.java [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
     InspectContainerCmd withContainerId(@Nonnull String containerId);
 
+    InspectContainerCmd withSize(Boolean showSize);
+
+    boolean getSize();
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/InspectContainerCmd.java]: should be object?

File: src/main/java/com/github/dockerjava/api/command/InspectContainerCmd.java [hunk 2]
```diff
@@ -12,6 +12,10 @@
 
     InspectContainerCmd withContainerId(@Nonnull String containerId);
 
+    InspectContainerCmd withSize(Boolean showSize);
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/InspectContainerCmd.java]: or then primitive here

File: src/main/java/com/github/dockerjava/api/command/InspectContainerCmd.java (hunk 1 above)
Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/api/command/InspectContainerCmd.java]: Fixed while merge.

**Pull request merged**